ScienceJournal/UI/TrialDetailViewController.swift

```diff
@@ -1002,7 +1002,18 @@ class TrialDetailViewController: MaterialHeaderViewController,
 
   // MARK: - ImageSelectorDelegate
 
-  func imageSelectorDidCreateImageData(_ imageData: Data, metadata: NSDictionary?) {
+  func imageSelectorDidCreateImageData(_ imageDatas: [ImageData]) {
+    guard let imageDataTuple = imageDatas.first else {
+      fatalError("Must have at least 1 image for [TrialDetailViewController].")
+    }
+
+    if imageDatas.count > 1 {
+      sjlog_info("[TrialDetailViewController] More than 1 ImageData.", category: .general)
+    }
+
+    let imageData = imageDataTuple.imageData
+    let metadata = imageDataTuple.metadata
```

Comment:
Since you're only referencing these one, I don't know if it's worth creating constants here.

Reply:
I was also thinking the same 👍